dnf-behave-tests/dnf/plugins-core/reposync.feature

```diff
@@ -260,11 +268,10 @@ Scenario: Reposync --urls and --download-metadata switches
     """
     And stdout matches line by line
     """
-    http://localhost:[0-9]+/repodata/primary.xml.gz
-    http://localhost:[0-9]+/repodata/filelists.xml.gz
-    http://localhost:[0-9]+/repodata/other.xml.gz
-    http://localhost:[0-9]+/repodata/comps.xml
-    http://localhost:[0-9]+/repodata/comps.xml.gz
+    http://localhost:[0-9]+/repodata/primary.xml.zst
+    http://localhost:[0-9]+/repodata/filelists.xml.zst
+    http://localhost:[0-9]+/repodata/other.xml.zst
+    http://localhost:[0-9]+/repodata/comps.xml.zst
```

Comment:
I believe the order has changed for this test and the one above since it uses the `std::map`.

Reply:
Yes, fixed.